### ISSUE TITLE
0.110 & dashboard compatibility & (TypeError: Cannot read property 'lovelace' of null)

### ISSUE THREAD

**Issue by iptvcld** [maykar/lovelace-swipe-navigation#21]
Logger: frontend.js.latest.202003181
Source: components/system_log/__init__.py:209
First occurred: 10:40:46 PM (4 occurrences)
Last logged: 10:42:55 PM

https://URL/local/community/lovelace-swipe-navigation/swipe-navigation.js:10:21 Uncaught TypeError: Cannot read property 'lovelace' of null
https://URL/local/community/custom-header/custom-header.js:1:7166 Uncaught TypeError: Cannot read property 'querySelector' of undefined

- url: /local/community/lovelace-swipe-navigation/swipe-navigation.js
   type: js

**Comment by Pirol62** [comment]
In the ressources view of the dashboard, the swipe plugin is marked as outdated
![image](https://user-images.githubusercontent.com/48820003/77904253-89b7e000-7284-11ea-9c84-6ea2282f6ed9.png)

**Comment by maykar** [comment]
Swipe-Navigation is next on my list for 0.107 compatibility. Right now Custom Header is my priority, but it seems like things are settling down there. Will be onto this soon.

**Comment by Pirol62** [comment]
I'm not shure, if it is related to this one, but I post it here first.
This error comes often:
````
Logger: frontend.js.latest.202003181
Source: components/system_log/__init__.py:209
First occurred: 6. April 2020, 22:59:04 (2 occurrences)
Last logged: 7:33:10

https://nauen.duckdns.org:8123/hacsfiles/lovelace-swipe-navigation/swipe-navigation.js:34:36 Uncaught TypeError: Cannot read property 'querySelectorAll' of null
````

Must have been appered when using one of my mobile devices

**Comment by Pirol62** [comment]
Update:
Receivd this error today:
````
Protokolldetails ( ERROR )
Logger: frontend.js.latest.202003181
Source: components/system_log/__init__.py:209
First occurred: 11:27:02 (37 occurrences)
Last logged: 12:59:48

https://nauen.duckdns.org:8123/hacsfiles/lovelace-swipe-navigation/swipe-navigation.js:10:21 Uncaught TypeError: Cannot read property 'lovelace' of null
https://xxx.ui.nabu.casa/hacsfiles/custom-header/custom-header.js:603:15224 Uncaught TypeError: Cannot read property 'setAttribute' of null
https://nauen.duckdns.org:8123/hacsfiles/custom-header/custom-header.js:1:15191 Uncaught TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.
````

**Comment by DavidFW1960** [comment]
This seems broken with 0.110.4 - not working at all

**Comment by maykar** [comment]
[I replied over on the forums](https://community.home-assistant.io/t/lovelace-swipe-navigation/113806/106), but is this related to this original issue/error? Any other info you could provide? This is still working as it was before 0.110.4 for me. A bit broken, but working as expected on the main dashboard.

**Comment by DavidFW1960** [comment]
On main dashboard, not working at all on my iPad.

**Comment by maykar** [comment]
As posted on the forums, DavidFW1960's issue was unrelated to this issue. He had a caching issue after an HA update that was eventually resolved with repeated clearings of the cache.

**Comment by liamstears** [comment]
Swipe navigation seems to be working but I also have this error?

`hacsfiles/lovelace-swipe-navigation/swipe-navigation.js:10:21 Uncaught TypeError: Cannot read property 'lovelace' of `null``

**Comment by maykar** [comment]
Reopened as this has just been pulled, but not released. I want to try to address one or two of the other issues before full release.

**Comment by maykar** [comment]
This fix has now been released in 1.2.4. Swiping should now work on all dashboards and you should no longer get the Lovelace TypeError.
